packages/cli-lib/lib/walk.js

```diff
@@ -7,6 +7,13 @@ const {
   STAT_TYPES,
 } = require('./read');
 
+const listFilesInDir = dir => {
```

Comment:
Is this similar to what the read func in read.js is doing?

Reply:
It's very similar, but it does it synchronously and just returns file names in the directory. I put it here because `walk()` does a very similar thing to it as well - but could move to read.js if you think it's better suited.